### ISSUE TITLE
Trace analytics

### ISSUE THREAD

**Issue by fbjork** [spandex-project/spandex_datadog#19]
How I enable trace & search analytics?

**Comment by GregMefford** [comment]
There is a configuration change you need to make in the Datadog Agent configuration to specify which traces should be analyzed. Did you already do that? I haven’t personally been using TS&A so there might be something broken, but I believe we had it working at one time in the past.

**Comment by GregMefford** [comment]
It could actually be that this is the same as #18 and it doesn’t currently work. I’m not sure when I can get some time to investigate more and try to get this working, but I would definitely like for it to work!

**Comment by fbjork** [comment]
@GregMefford Yes, the agent has been configured to send trace analytics. It works for our Ruby using ddtrace-rb, but not with spandex from what I can tell.

**Comment by san650** [comment]
Looks like the support was added in https://github.com/spandex-project/spandex_datadog/pull/23
